docs/installation/building-from-source.rst

```diff
@@ -92,10 +92,9 @@ Many of Pillow's features require external libraries:
 * **libavif** provides support for the AVIF format.
 
   * Pillow requires libavif version **1.0.0** or greater.
-  * libavif is merely an API that wraps AVIF codecs. If you are compiling
-    libavif from source, you will also need to install both an AVIF encoder
-    and decoder, such as rav1e and dav1d, or libaom, which both encodes and
-    decodes AVIF images.
+  * libavif is merely an API that wraps AVIF codecs. If you are compiling libavif from
+    source, you will also need to install libaom, which both encodes and decodes AVIF
+    images.
```

Comment:
We should only consider removing `dav1d` as a last resort. Although the decoder in libaom is the reference implementation and we test it with open-source and commercial AV1 test vectors and fuzzing regularly at Google, dav1d is likely to have better software security because it is being used in far more applications, including some that perform additional fuzzing of dav1d.